WordPress/src/main/java/org/wordpress/android/ui/reader/discover/ReaderPostUiStateBuilder.kt

```diff
@@ -230,14 +231,16 @@ class ReaderPostUiStateBuilder @Inject constructor(
         post: ReaderPost,
         onBlogSectionClicked: (Long, Long) -> Unit,
         postListType: ReaderPostListType? = null,
-        isP2Post: Boolean = false
-    ) = buildBlogSectionUiState(post, onBlogSectionClicked, postListType, isP2Post)
+        isP2Post: Boolean = false,
+        isReaderImprovementsEnabled: Boolean = false
```

Comment:
Nitpick: would it make sense to add a [trailing comma](https://kotlinlang.org/docs/coding-conventions.html#trailing-commas) here?